src/net/util/show_packet.h

```diff
@@ -20,7 +20,7 @@ static inline void show_packet(uint8_t *raw, int size, char *title) {
 	if (!&mod_logger)
 		return;
 
-	if (&mod_logger.logging &&
+	if (//&mod_logger.logging &&
```

Comment:
This seems as inlined version of `logging_raw` function, which actually checks for this address. I'd suggest to totally remove commented line and replace also runtime check for `logging` in `logging_raw` with assert

We could also ask author (@vloginova) for better way